openedx/core/djangoapps/user_authn/views/logout.py

```diff
@@ -83,6 +84,17 @@ def dispatch(self, request, *args, **kwargs):
         delete_logged_in_cookies(response)
 
         mark_user_change_as_expected(None)
+
+        # Redirect to tpa_logout_url if TPA_AUTOMATIC_LOGOUT_ENABLED is set to True and if
+        # tpa_logout_url is configured.
+        #
+        # NOTE: This step skips rendering logout.html, which is used to log the user out from the
+        # different IDAs. To ensure the user is logged out of all the IDAs be sure to redirect
+        # back to <LMS>/logout after logging out of the TPA.
```

Comment:
We should include this note in the flag documentation and highlight it if possible.

Reply:
Added a toggle warning. Thanks for the suggestion.